recipe/meta.yaml

```
@@ -2,6 +2,7 @@
 {% set version = "1.1dev" %}
 {% set build_number = 34 %}
 {% set build_number = build_number + 500 %}  # [FEATURE_DEBUG]
+{% set sha256 = 65dc32f217a3fe757f5b81bb45dae0cc151c82304e8016ddcc9efbd46fbbc864 %}
```

```suggestion
{% set sha256 = "65dc32f217a3fe757f5b81bb45dae0cc151c82304e8016ddcc9efbd46fbbc864" %}
```

ci failure says

> expected token 'end of statement block', got 'dc32f217a3fe757f5b81bb45dae0cc151c82304e8016ddcc9efbd46fbbc864'

which leads me to believe it is trying to take it as a number since the error starts with the first letter